channels/email/email-headers.rst

```diff
@@ -158,3 +159,46 @@ email articles.
 ``X-Zammad-Ignore``
    | Tell Zammad to silently drop the Email.
    | Example: ``X-Zammad-Ignore: true``
+
+Trusted Channel
+---------------
+.. note:: **🚧 Self Hosted only 🚧**
+
+   Below settings are only available to self hosted users.
+
+.. danger::
+   ⚠️ As stated above, this is dangerous and can lead to unexpected behavior in
+   the communication with external parties. Only follow the instructions below,
+   if you know what you are doing.
+
+Setting a channel to ``trusted`` can ony be done via `console`_. In the
+rails console, execute the following commands:
+
+| List all channels in Zammad:
+
+.. code-block:: ruby
+
+   >> Channel.all
+
+Look for the ``id`` of the channel, you want to set to ``trusted``.
+
+Select your identified channel (replace the 99 with the correct id):
+
+.. code-block:: ruby
+
+   >> channel = Channel.find(99)
+
+Show the currently acivated opions of the selected channel:
+
+.. code-block:: ruby
+
+   >> options = channel[:options]
+
+Add the ``"trusted"=>true`` flag for the inbound part of the channel:
+
+.. code-block:: ruby
+
+   >> options[:inbound][:trusted] = true
```

Comment:
This looks good, but I believe that the settings will not be saved. Try it by reopening the Rails console and checking again.
`channel.save!` should do the trick.

Reply:
Thank you! Worked :-)